client/app/visualizations/chart/plotly.js

```diff
@@ -270,11 +270,15 @@ const PlotlyChart = () => {
               },
             };
 
+            console.log(scope.options.seriesOptions);
```

Comment:
I am sure `the totally not annoying I swear` linter will remind you to remote this 😁